sphere-models/src/it/java/io/sphere/sdk/taxcategories/TaxCategoryFixtures.java

```diff
@@ -57,4 +58,13 @@ public static void withTaxCategory(final TestClient client, final TaxCategoryDra
         user.accept(taxCategory);
         client.execute(TaxCategoryDeleteCommand.of(taxCategory));
     }
+
+    public static void withUpdateableTaxCategory(final TestClient client, final UnaryOperator<TaxCategory> user) {
```

Comment:
Is `user` a good parameter name for that function? Isn't it more something like `test`?